CharacterRandomizer.js

```diff
@@ -0,0 +1,112 @@
+import React, { useState, useEffect } from 'react';
+import { Text, View } from 'react-native';
+import styled from 'styled-components/native';
+
+//Note! Some comments are in Swedish, beacuse this app is mainly focused on swedish rpg.
+
+//The Character-randomizer component
+
+const CharacterRandomizer = () => {
+	const [characterName, setCharacterName] = useState([]);
+
+	//re-used the random selector from my cluedo-project. No need to reinvent the wheel :)
+	const randomSelector = (array) => {
+		return array[Math.floor(Math.random() * array.length)];
+	};
+
+	//sadly I could not find a api with only swedish names
+	const API_URL =
+		'https://randomapi.com/api/6de6abfedb24f889e0b5f675edc50deb?fmt=raw&sole';
+
+	useEffect(() => {
+		fetch(API_URL)
+			.then((res) => res.json())
+			.then((names) => {
+				const randomName = randomSelector(names);
+				setCharacterName(randomName);
+			});
+	}, []);
+
+	//folkslag
+	const races = [
+		'Human',
+		'Insectoid',
+		'Mutant',
+		'Rese',
+		'Robot',
+		'Smallpeople',
+		'Underground',
+		'Ancient',
+	];
+
+	const randomRace = () => randomSelector(races);
+
+	//can't use class because it's a reserved keyword
+	const klass = ['Warrior', 'Magician', 'Thief'];
+
+	const randomKlass = () => randomSelector(klass);
+	//ödesmakt
+	const faith = ['Order', 'Chaos', 'Balance'];
+
+	const randomFaith = () => randomSelector(faith);
+	//styrka
+	const strenght = Math.floor(Math.random() * 18) + 3;
```

Comment:
`Math.floor(Math.random() * 18) + 3;` should be moved to separate function since we use it multiple times